viewing_party/main.py

```diff
@@ -0,0 +1,272 @@
+# <-------------- WAVE 1 ------------------ WAVE 1 ------------------------- >
+
+# test wave 1, part 1:
+# create function: create_movie()
+# function takes 2 parameters (title, genre, rating)
+# IF title, genre and rating == true ==> return a dictionary
+# ELSE (title, genre and rating == false ) ==> return None
+
+def create_movie(title, genre, rating):
+    movies = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if title and genre and rating:
+        print(movies)
+        return movies
+    else:
+        return None
+
+# test wave 1, part 2:
+# create function: add_to_watch()
+# function takes 2 parameters (user_data, movie)
+# user_data ==> dictionary: key = watched; value = []
+    # represents that user has NO movies in watched list
+# movie ==> dictionary w/ 3 keys: title, genre, rating
+# add movie to watched list inside of user_data
+# return user_data
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+# test wave 1, part 3:
+# SAME AS PART 2, DIFF VARIABLES
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+# test wave 1, part 4:
+# create function watch_movie
+# function takes 2 parameters: user_data, title
+# user_data ==> dictionary: key: watchlist, watched; value: []
+# title ==> string (title of movie user has watched)
+# IF title of movie is IN user's watchlist:
+    # remove movie from WATCHLIST
+    # ADD movie to WATCHED
+    # return user_data
+# IF title of movie IS NOT IN user's watchlist:
+    # return user_data
```

Comment:
I love that you wrote pseudocode first! That said, its usually preferred that you remove pseudocode comments like this before committing and/or submitting your code.